roles/chart_verifier/tasks/tests.yml

```diff
@@ -90,29 +90,25 @@
         - chart_stat is defined
         - chart_stat.stat.exists
 
-    - name: "Check values_file stat"
+    - name: "Check if chart.values_file exist"
       stat:
-        path: "{{ chart.values_file }}"
-      register: values_stat
-      when:
```

Comment:
Also, I guess we should keep the "when" condition here as long as the values_file variable is optional.

Reply:
This block is only is applied only for local files. See line 113.